topcoffea/modules/utils.py

```diff
@@ -113,6 +113,8 @@ def read_cfg_file(fpath,cfg={},max_files=0):
                 # Note: This implicitly assumes that a redirector line will appear before any json
                 #   paths in the cfg file
                 xrd_src = l
+            if l.startswith("file://"):
```

Comment:
Hmm, I might just be thinking about this wrong, but should this be an `elif`?

Reply:
Ah, yes you are correct. I think logically it does not matter much as the statement can't be true in the two tests, but that was just luck.